security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationResources.scala

```diff
@@ -13,8 +14,13 @@ trait AuthenticationResources extends Directives with FailFastCirceSupport {
   val name: String
   val frontendStrategySettings: FrontendStrategySettings
 
+  // TODO: deprecated - it will be removed when all endpoints are moved to Tapir
```

Comment:
Let's add some more descriptive comment, about why we have to provide three methods, that we currently support two authentication methods (via Akka and Tapir) and one can't be adapted by the second. Also, I'm not sure if I understand correctly why we have two instead of one additional method. Am I right that this `authenticationMethod` is only a kind of definition and `authenticate` is part of server logic?

Reply:
yes, correct. Notice that Tapir doesn't bring you the auth method implementations. You have to use your own or fallback to the server's one you use in Tapir's endpoints interpreter.

Reply:
Nice, thanks